### PR TITLE
regendoc: reduce version noise by replacing minor/patch with placeholders

### PR DESCRIPTION
**Pull request by RonnyPfannschmidt**

### CODE REVIEW

File: doc/en/example/parametrize.rst [hunk 1]
```diff
@@ -397,10 +397,32 @@ is to be run with different sets of arguments for its three arguments:
...
+   KeyError: local('$REGENDOC_TMPDIR/CWD/conftest.py')
+   
+   During handling of the above exception, another exception occurred:
+   Traceback (most recent call last):
+     File "$PYTHON_PREFIX/lib/python3.5/site-packages/_pytest/config.py", line 363, in _importconftest
+       mod = conftestpath.pyimport()
+     File "$PYTHON_PREFIX/lib/python3.5/site-packages/py/_path/local.py", line 680, in pyimport
+       raise self.ImportMismatchError(modname, modfile, self)
+   py._path.local.LocalPath.ImportMismatchError: ('conftest', '$PWD/example/conftest.py', local('$REGENDOC_TMPDIR/CWD/conftest.py'))
+   ERROR: could not load $REGENDOC_TMPDIR/CWD/conftest.py
```
Comment (The-Compiler) [inline doc/en/example/parametrize.rst]: Looks like something broke?

File: doc/en/example/pythoncollection.rst [hunk 2]
```diff
@@ -162,18 +162,32 @@ Finding out what is collected
...
+    KeyError: local('$REGENDOC_TMPDIR/CWD/conftest.py')
+    
+    During handling of the above exception, another exception occurred:
+    Traceback (most recent call last):
+      File "$PYTHON_PREFIX/lib/python3.5/site-packages/_pytest/config.py", line 363, in _importconftest
+        mod = conftestpath.pyimport()
+      File "$PYTHON_PREFIX/lib/python3.5/site-packages/py/_path/local.py", line 680, in pyimport
+        raise self.ImportMismatchError(modname, modfile, self)
+    py._path.local.LocalPath.ImportMismatchError: ('conftest', '$PWD/example/conftest.py', local('$REGENDOC_TMPDIR/CWD/conftest.py'))
+    ERROR: could not load $REGENDOC_TMPDIR/CWD/conftest.py
```
Comment (The-Compiler) [inline doc/en/example/pythoncollection.rst]: same

File: doc/en/getting-started.rst [hunk 3]
```diff
@@ -26,7 +26,7 @@ Installation::
 To check your installation has installed the correct version::
 
     $ pytest --version
-    This is pytest version 3.0.7, imported from $PYTHON_PREFIX/lib/python3.5/site-packages/pytest.py
+    This is pytest version 3.0.8.dev260+gb6125d9.d20170512, imported from $PYTHON_PREFIX/lib/python3.5/site-packages/pytest.py
```
Comment (The-Compiler) [inline doc/en/getting-started.rst]: not replaced correctly

File: doc/en/skipping.rst [hunk 4]
```diff
@@ -223,27 +223,32 @@ Here is a simple test file with the several usages:
...
+    KeyError: local('$REGENDOC_TMPDIR/example/conftest.py')
+    
+    During handling of the above exception, another exception occurred:
+    Traceback (most recent call last):
+      File "$PYTHON_PREFIX/lib/python3.5/site-packages/_pytest/config.py", line 363, in _importconftest
+        mod = conftestpath.pyimport()
+      File "$PYTHON_PREFIX/lib/python3.5/site-packages/py/_path/local.py", line 680, in pyimport
+        raise self.ImportMismatchError(modname, modfile, self)
+    py._path.local.LocalPath.ImportMismatchError: ('conftest', '$PWD/example/conftest.py', local('$REGENDOC_TMPDIR/example/conftest.py'))
+    ERROR: could not load $REGENDOC_TMPDIR/example/conftest.py
```
Comment (The-Compiler) [inline doc/en/skipping.rst]: ...and more breakage

File: doc/en/warnings.rst [hunk 5]
```diff
@@ -21,35 +21,28 @@ and displays them at the end of the session::
...
-
+    
     -- Docs: http://doc.pytest.org/en/latest/warnings.html
-    1 passed, 1 warnings in 0.01 seconds
+    ======= 1 passed, 1 warnings in 0.12 seconds ========
 
 The ``-W`` flag can be passed to control which warnings will be displayed or even turn
 them into errors::
 
     $ pytest -q test_show_warning.py -W error::DeprecationWarning
```
Comment (nicoddemus) [inline doc/en/warnings.rst]: This should be `test_show_warnings.py` instead.

File: doc/en/unittest.rst [hunk 6]
```diff
@@ -183,7 +183,13 @@ Running this test module ...::
 
     $ pytest -q test_unittest_cleandir.py
     .
-    1 passed in 0.12 seconds
+    ======= warnings summary ========
+    test_unittest_cleandir.py::MyTest::test_method
+      $REGENDOC_TMPDIR/test_unittest_cleandir.py:11: ResourceWarning: unclosed file <_io.TextIOWrapper name='samplefile.ini' mode='r' encoding='UTF-8'>
+        s = open("samplefile.ini").read()
+    
+    -- Docs: http://doc.pytest.org/en/latest/warnings.html
```
Comment (The-Compiler) [inline doc/en/unittest.rst]: Oh. We should probably suppress (or fix) warnings in the doc output if irrelevant...

File: doc/en/unittest.rst [hunk 7]
```diff
@@ -183,7 +183,13 @@ Running this test module ...::
 
     $ pytest -q test_unittest_cleandir.py
     .
-    1 passed in 0.12 seconds
+    ======= warnings summary ========
+    test_unittest_cleandir.py::MyTest::test_method
+      $REGENDOC_TMPDIR/test_unittest_cleandir.py:11: ResourceWarning: unclosed file <_io.TextIOWrapper name='samplefile.ini' mode='r' encoding='UTF-8'>
```
Comment (nicoddemus) [inline doc/en/unittest.rst]: Hmm this warning doesn't seem to be right

File: doc/en/skipping.rst [hunk 8]
```diff
@@ -223,27 +223,32 @@ Here is a simple test file with the several usages:
...
-    XFAIL xfail_demo.py::test_hello3
-      condition: hasattr(os, 'sep')
-    XFAIL xfail_demo.py::test_hello4
-      bug 110
-    XFAIL xfail_demo.py::test_hello5
-      condition: pytest.__version__[0] != "17"
-    XFAIL xfail_demo.py::test_hello6
-      reason: reason
-    XFAIL xfail_demo.py::test_hello7
+    During handling of the above exception, another exception occurred:
```
Comment (nicoddemus) [inline doc/en/skipping.rst]: This doesn't seem to be right

File: doc/en/example/parametrize.rst (hunk 1 above)
Comment (RonnyPfannschmidt) [inline doc/en/example/parametrize.rst]: good catch

File: doc/en/getting-started.rst (hunk 3 above)
Comment (RonnyPfannschmidt) [inline doc/en/getting-started.rst]: nice find, that one is completely different to match, i will take a look

File: doc/en/unittest.rst (hunk 6 above)
Comment (nicoddemus) [inline doc/en/unittest.rst]: Yep, the test code should be fixed to use a `with` statement to open the file.

**Comment by nicoddemus** [comment]
If you could fix up the last two remaining issues it would be great @RonnyPfannschmidt!

**Comment by nicoddemus** [comment]
I will fix the small remaining issues myself, thanks @RonnyPfannschmidt!

**Pull request merged**